src/components/alert/alert.stories.mdx

```diff
@@ -66,10 +66,67 @@ Alerts work well even when displayed at full width. Their contents will constrai
   </Story>
 </Canvas>
 
+## Icon
+
+An option to add an icon to the alert. You can add any icon name from our library in the `icon_name` property. Right now, the color is hardcoded to orange to satisfy the current use case of our offline notification, but it would be nice to add more color options in the future.
+
+<Canvas>
+  <Story
+    name="Icon"
+    args={{
+      message: 'Your action was completed successfully! 🎉',
+      icon_name: 'check',
+    }}
+  >
+    {(args) => template(args)}
+  </Story>
+</Canvas>
+
+## Floating
+
+By adding `floating: true`, a light border and shadow will be added to the alert for a floating effect. The background is white to improve the contrast between the border, shadow and background. In the future, we could make it a possibility to change the background color by itself.
+
+<Canvas>
+  <Story
+    name="Floating"
+    args={{
+      message: 'You appear to be offline. 🤔',
+      dismissable: true,
+      floating: true,
+      icon_name: 'bell',
+    }}
+  >
+    {(args) => template(args)}
+  </Story>
+</Canvas>
+
+When more contrast is desired, you may also attach a theme class to the the floating variation. In this example, the card has a class of `t-light` within a `t-dark` container:
+
+<Canvas>
+  <Story
+    name="Themed Floating"
+    parameters={{
+      theme: 't-dark',
+    }}
+    args={{
+      message: 'You appear to be offline. 🤔',
+      dismissable: true,
+      floating: true,
+      icon_name: 'bell',
+      class: 't-light',
+    }}
+  >
+    {(args) => template(args)}
+  </Story>
+</Canvas>
+
 ## Template Properties
 
 - `class` (string)
+- `icon_name` (string)
```

Comment:
Any reason not to call this simply `icon`? (Property names and block names don't conflict.)